packages/user-interface/src/pages/CasesPageRequests.mock.tsx

```diff
@@ -0,0 +1,92 @@
+import { FetchResult, GraphQLRequest } from "@apollo/client";
+import { QUERY_GET_ZAKEN } from "@nl-portal/nl-portal-api";
+
+const getZaken: {
+  request: GraphQLRequest;
+  result: FetchResult<Record<string, any>>;
+} = {
+  request: {
+    query: QUERY_GET_ZAKEN,
+    variables: {},
+  },
+  result: {
+    data: {
+      getZaken: [
```

Comment:
Hiervoor geldt hetzelfde als voor wat ik eerder noemde. Ik zou dan eerder de zaakidentificaties hernoemen naar iets wat duidelijk een voorbeeld/mock zaaktype is, zodat het niet meer gelinkt aan iets van Den Haag.

Reply:
yep, ik begrijp het. Misschien kan ik in de titel gewoon 'open-zaak' en 'gesloten-zaak' gebruiken en dat testen. In principe gaat het er alleen om dat dit veld op het scherm terecht komt toch?